src/wovnio/html/HtmlConverter.php

```diff
@@ -273,7 +273,11 @@ private function buildHrefLang($lang_code)
             $url = $this->headers->removeLang($url, $this->store->defaultLang());
             $url = Url::addLangCode($url, $this->store, $lang_code, $this->headers);
         } else {
-            if ($lang_code !== $this->store->defaultLang()) {
+            $customDomainLangs = $this->store->getCustomDomainLangs();
+            $customDomainLang = $customDomainLangs ? $customDomainLangs->getCustomDomainLangByLang($lang_code) : false;
```

Comment:
I feel strange about that `$customDomainLang` can have a different type of variables like boolean and customDomainLang.

Reply:
I prefer that.